mobile/src/common/types/notification/notification-service.type.ts

```diff
@@ -0,0 +1,9 @@
+import { NotificationType } from '~/common/enums/enums';
+
+type ShowNotificationType = {
```

Comment:
We store in src/common/types only common types, all specific types we store in same place, move this to services/notifications/common/types/show-notification-params